draft-west-cookie-incrementalism.md

```diff
@@ -401,85 +404,361 @@ from "have the same registrable domain" (or "is an exact match for") to say
 
 Note: The request's URL when establishing a WebSockets connection has scheme "http" or "https", rather than "ws" or "wss". FETCH maps schemes when constructing the request. This mapping allows same-site cookies to be sent with WebSockets.
 
-## Scheming Cookies {#scheming-cookies}
+## Origin-Bound Cookies {#origin-bound-cookies}
+
+Cookies are one of the few components of the web platform that are not
+scoped to the origin by default. This difference in scoping means that
+cookies have weaken confidentiality and integrity compared with other
+storage APIs on the web platform.
+
+Examples:
+
+1. `https://somesite.com` sets a simple cookie, `secret=123456`,
+    which contains private information about a user. Information that
```

Comment:
Style nit: The body of each of these items is offset from the first line by one character. Is that intentional?